docs/source/tutorial_projection.md

```diff
@@ -143,17 +143,17 @@ L(u(\mu); \mu) = F(\mu)
 
 Here, {math}`L` is a linear or non-linear parametric {{ Operator }} and {math}`F` is a
 parametric right-hand side vector. In {{ StationaryModel }}, {math}`L` is represented by
-the {attr}`~pymor.models.basic.StationaryModel.operator` attribute. So
+the {attr}`!~pymor.models.basic.StationaryModel.operator` attribute. So
```

Comment:
Hmm, this is an attribute, that is documented. Seems like we cannot link any attributes at all? I guess this should somehow (later) be fixed.

Reply:
I guess it's because `StationaryModel` has a "Parameters" section, but not an "Attributes" section.

Reply:
Ah ok. So we would need to document all `__init__` args of `ImmutableObjects` twice. Maybe a better way would be to generally use 'Attributes' instead of 'Parameters'? (All args end up as attributes anyway.)